code/controllers/subsystem/ticker.dm

```diff
@@ -599,6 +599,9 @@ SUBSYSTEM_DEF(ticker)
 	if(delay_end && !skip_delay)
 		to_chat(world, "<span class='boldannounce'>An admin has delayed the round end.</span>")
 		return
+	if(GLOB.ahelp_tickets.ticketAmount && GLOB.admins.len) //YOGS - tickets
```

Comment:
Make it show the "theres still tickets but no admins, make a complaint on the forum" message.